### ISSUE TITLE
Microblocks

### ISSUE THREAD

**Issue by Matthysse** [Chisel-Team/Chisel#753]
Can you add a micro blocks support please? In 1.7.10 it worked but not in 1.12.2 more.

**Comment by ChloeDawn** [comment]
Which microblocks mod?

**Comment by Matthysse** [comment]
Forge Micro blocks from Chickencode

**Comment by Graywaren** [comment]
Agree! I've been trying to manually add chisel blocks to be compatible with no luck yet. Desperately want to be able to make all the different shapes/items that microblocks allows with chisel items.

**Comment by TheDeviantCrafter** [comment]
Forge Microblocks: Chicken Bones Edition is the most popular mod that doesn't have working compatibility. Little Tiles and Chisels and Bits should be compatible with any solid block out of the box.

**Comment by GrumpyPirate** [comment]
[This microblocks config](https://gist.github.com/GrumpyPirate/6275cefa7ccdaa853ea8436f043505b3) can be used to add support for Chisel blocks. On multiplayer, it needs to be applied both server- and client-side (for all players).
